Framework/include/QualityControl/TaskFactory.h

```diff
@@ -55,7 +55,7 @@ class TaskFactory
     std::string library = "lib" + taskConfig.moduleName;
     logger << "Loading library " << library << AliceO2::InfoLogger::InfoLogger::endm;
     int libLoaded = gSystem->Load(library.c_str(), "", true);
-    if (libLoaded) {
+    if (libLoaded < 0) {
```

Comment:
Good catch. How did it ever work ?

Reply:
Before, during the topology generation, we happened to load each library only once. It was no longer the case when I started to work on that complex topology example, then the bug revealed itself.